api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -131,18 +154,18 @@ def publish_cdr(cmd_name, args)
     # See https://docs.google.com/document/d/1EHw5nisXspJjA9yeZput3W4-vSIcuLBU5dPizTnk1i0/edit
     common.run_inline %W{bq mk -f --default_table_expiration 7200 --dataset #{ingest_dataset}}
     common.run_inline %W{./copy-bq-dataset.sh
-        #{source_dataset} #{ingest_dataset} #{env.fetch(:source_cdr_project)}
+        #{source_dataset} #{ingest_dataset} #{tier[:source_cdr_project]}
         #{table_match_filter} #{table_skip_filter}}
 
     common.run_inline %W{bq mk -f --dataset #{dest_dataset}}
     common.run_inline %W{./copy-bq-dataset.sh
-        #{ingest_dataset} #{dest_dataset} #{env.fetch(:ingest_cdr_project)}
+        #{ingest_dataset} #{dest_dataset} #{tier[:ingest_cdr_project]}
         #{table_match_filter} #{table_skip_filter}}
 
     # Delete the intermediate dataset.
-    common.run_inline %W{bq rm -rf --dataset #{ingest_dataset}}
+    common.run_inline %W{bq rm -r -f --dataset #{ingest_dataset}}
```

Comment:
I was playing around with a different version of `bq` and it didn't recognize the argument "rf" but it worked when they were separately specified